docs/migrating-from-asciidoc.adoc

```diff
@@ -0,0 +1,329 @@
+// Issue #480. #434, #463 refer.
+:ap: Asciidoc Python
+:ad: Asciidoctor
```

Comment:
Let's use "adr" for Asciidoctor, short for "asciidoc + doctor" and "adp" for "asciidoc python". That catches the syllables too.